src/index.js

```diff
@@ -31,7 +31,18 @@ async function createNewEventHandler(kc) {
       kubeClass: kc,
       logger: log,
       requestOptions: { qs: { timeoutSeconds: process.env.CRD_WATCH_TIMEOUT_SECONDS || 300 } },
-      livenessInterval: true
+      livenessInterval: true,
+      options: {
+        rr: {
+          reconcileByDefault: process.env.RR_RECONCILE_BY_DEFAULT || null
+        },
+        s3: {
+          reconcileByDefault: process.env.RR_S3_RECONCILE_BY_DEFAULT || null
+        },
+        git: {
+          reconcileByDefault: process.env.RR_GIT_RECONCILE_BY_DEFAULT || null
+        }
```

Comment:
lets keep it simple and just have one `RR_RECONCILE_BY_DEFAULT` for the whole controller. while i think this would work, i think its confusing and i doubt someone would be wanting different behavior for different types of RRs.